src/cli.ts

```diff
@@ -3309,19 +3294,19 @@ function decryptMnemonic(network: CLINetworkAdapter, args: string[]) : Promise<s
 
 /* Print out all documentation on usage in JSON 
  */
-interface DocsArgsType {
+type DocsArgsType = {
```

Comment:
Note: these were changed to `type` defs because `interface` defs don't play nicely with the stringify-able JSON type check.